README.md

```diff
@@ -9,6 +9,11 @@ Plug 'nvim-treesitter/nvim-treesitter', {'do': ':TSUpdate'}
 Plug 'theHamsta/nvim-dap-virtual-text'
 ```
 
+> [!NOTE]
+>
+> With Neovim 0.9 and above, `nvim-treesitter` is not a dependency.
+> This plugin only needs the parsers for the languages you want to use it with.
```

Comment:
@mrcjkb nvim-treesitter is mainly needed as source for the `locals.scm`. Of course you could get them also from elsewhere but the default experience won't be nice. I'll adapt the help text that nvim-treesitter is the main source of locals.scm